projects/driver_controls/inc/stm32f0xx/ads1015.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+// Module for using ads1015
```

Comment:
I'm not sure why you put the header in the stm32 folder - the header itself isn't platform-specific.